node/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# Build Stage
+FROM rust:1.72.1 as build
```

Comment:
any reason why you put docker file in the node/ while makefile lives in node/tools ?